PhysicsTools/NanoAOD/python/taus_updatedMVAIds_cff.py

```diff
@@ -355,194 +359,124 @@
-        ),
-        cms.PSet(
-            category = cms.uint32(15),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_EC_WPeff70'),
-            variable = cms.string('pt')
-        )
-    )
-)
+patTauDiscriminationByTightElectronRejectionMVA62018 = patTauDiscriminationByVLooseElectronRejectionMVA62018.clone()
+for m in patTauDiscriminationByTightElectronRejectionMVA62018.mapping:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff70")
 # VTight
-patTauDiscriminationByVTightElectronRejectionMVA62018 = patTauDiscriminationByVLooseElectronRejectionMVA62018.clone(
+patTauDiscriminationByVTightElectronRejectionMVA62018 = patTauDiscriminationByVLooseElectronRejectionMVA62018.clone()
+for m in patTauDiscriminationByVTightElectronRejectionMVA62018.mapping:
+    m.cut = m.cut.value().replace("WPeff98", "WPeff60")
+### Define v1 anit-e discriminants (2015)
+antiElectronDiscrMVA6_version = "MVA6v1"
+## Raw
+patTauDiscriminationByElectronRejectionMVA62015Raw = patTauDiscriminationAgainstElectronMVA6.clone(
+    Prediscriminants = noPrediscriminants, #already selected for MiniAOD
+    vetoEcalCracks = True, #don't keep tau candidates in EB-EE cracks for v1
+    mvaName_NoEleMatch_wGwoGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_wGwoGSF_BL',
+    mvaName_NoEleMatch_wGwoGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_wGwoGSF_EC',
+    mvaName_NoEleMatch_woGwoGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_BL',
+    mvaName_NoEleMatch_woGwoGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_EC',
+    mvaName_wGwGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_BL',
+    mvaName_wGwGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_wGwGSF_EC',
+    mvaName_woGwGSF_BL = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_BL',
+    mvaName_woGwGSF_EC = 'RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_woGwGSF_EC'
+)
+## anti-e v1 WPs
+# VLoose
+patTauDiscriminationByVLooseElectronRejectionMVA62015 = patTauDiscriminantCutMultiplexer.clone(
+    PATTauProducer = patTauDiscriminationByElectronRejectionMVA62015Raw.PATTauProducer,
+    Prediscriminants = patTauDiscriminationByElectronRejectionMVA62015Raw.Prediscriminants,
+    toMultiplex = cms.InputTag("patTauDiscriminationByElectronRejectionMVA62015Raw"),
+    key = cms.InputTag("patTauDiscriminationByElectronRejectionMVA62015Raw","category"),
     mapping = cms.VPSet(
         cms.PSet(
             category = cms.uint32(0),
-            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_BL_WPeff60'),
+            cut = cms.string('RecoTauTag_antiElectron'+antiElectronDiscrMVA6_version+'_gbr_NoEleMatch_woGwoGSF_BL_WPEff99'),
             variable = cms.string('pt')
```

Comment:
it looks like the mapping values are identical to what already exists in this file in patTauDiscriminationByVLooseElectronRejectionMVA62018 with 98 -> 98.
It would be nice to respect this similarity and use a similar string replace

Reply:
Ok, I have added this in the latest commit